Sources/SwiftFormatRules/DeclSyntaxProtocol+Comments.swift

```diff
@@ -62,7 +62,7 @@ extension DeclSyntaxProtocol {
       case .newlines(let n), .carriageReturns(let n), .carriageReturnLineFeeds(let n):
         // Only allow for 1 newline between doc line comments, but allow for newlines between the
         // doc comment and the declaration.
-        guard n == 1 || !hasSeenFirstLineComment else { break gatherComments }
+        guard !hasSeenFirstLineComment else { break gatherComments }
```

Comment:
Why is this change necessary?

Reply:
It forces us to have a new line before `- parameters`, but if it is before `- parameter` it does not require.